CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `TerminalComponentModeler` defaults to the pseudo wave definition of scattering parameters. The new field `s_param_def` can be used to switch between either pseudo or power wave definitions.
 - Restructured the smatrix plugin with backwards-incompatible changes for a more robust architecture. Notably, `ComponentModeler` has been renamed to `ModalComponentModeler` and internal web API methods have been removed. Please see our migration guide for details on updating your workflows.
 - Prevent small bandwidth sources from being created in `TerminalComponentModeler` when modeler frequencies are close together.
+- Added error for when a user is specifying `element_mappings` and using autograd with a `ComponentModeler`.
```

Comment:
I'd argue we dont need this because we never supposed using autograd with component modeler in the first place, so there's probably no need for it to be documented that we stopped supporting it.